src/stores/AdminStore.js

```diff
@@ -51,6 +57,15 @@ export const AdminStore = Fluxxor.createStore({
         return this.dataStore;
     },
 
+    loadColumns(columnValues, saveAsColumnName) {
+      let columns = [];
+      columnValues.forEach(value => {
+        columns.push(Object.assign({}, DEFAULT_COLUMN, value));
+      });
```

Comment:
Nit: Given that columnValues is a list, this can be done in one line using map:

```js
const columns = columnValues.map(value => Object.assign({}, DEFAULT_COLUMN, value));
```

Reply:
ok!